app/assets/javascripts/components/components/status_content.jsx

```diff
@@ -93,7 +93,7 @@ const StatusContent = React.createClass({
     const { status } = this.props;
     const { hidden } = this.state;
 
-    const content = { __html: emojify(status.get('content')) };
+    const content = { __html: emojify(status.get('content').replace(/\s*(<p>|<\/p>)\s*/g, '$1').replace(/\s{2}/g, ' &nbsp;')) };
```

Comment:
I know how non-breaking space works and why you only replace *starting* with two consecutive spaces. But you are replacing *two* spaces with *one* non-breaking.

I'm not seeing why stripping out `<p>` tags is necessary in this place, even if they're there. It's either the `<br />` from your version, or the `<p>` from older versions, but both should work the same, especially since we don't separate HTML tags with newlines there shouldn't be anything "extra" to strip out.